syncstorage/storage/sql/dbconnect.py

```diff
@@ -371,6 +454,11 @@ def get_query(self, name, params):
                 qvars["bso"] = params["bso"]
             else:
                 qvars["bso"] = self.get_bso_table(params["userid"])
+        if "%(batch_upload_items)s" in query:
```

Comment:
nit: the inconsistency between abbreviated `%(bso)s` and long-form `%(batch_upload_items)s` is a little strange.